lib/editor/components/EntityDetailsHeader.js

```diff
@@ -199,11 +203,11 @@ class EntityDetailsHeader extends Component<Props> {
                 ? <span className='fa-stack'>
                   <Icon
                     type='square'
-                    style={{color: `#${activeEntity.route_color || 'fff'}`}}
+                    style={{color: `#${typeof activeEntity.route_color === 'string' ? activeEntity.route_color : 'fff'}`}}
```

Comment:
Shouldn't this default be dealt with on the server?

Reply:
Potentially. These changes are all just issues that got exposed when we changed the Entity type. I think moving the default to the server is out of scope for this PR, but we could do that in another one. However, there's precedent for having defaults on the client, like with pickup/dropoff types, etc.